packages/xo-server/src/api/host.js

```diff
@@ -265,3 +265,19 @@ installSupplementalPack.params = {
 installSupplementalPack.resolve = {
   host: ['host', 'host', 'admin'],
 }
+
+// -------------------------------------------------------------------
+
+export function isHyperThreading({ host }) {
+  return this.getXapi(host).isHyperThreading(host._xapiId)
+}
+
+isHyperThreading.description = 'get hyperthreading information'
```

Comment:
hyper-threading ?

Reply:
`hyper-threading` seems good to my :) thank you